internal/internal_workflow_testsuite.go

```diff
@@ -1296,10 +1300,7 @@ func (env *testWorkflowEnvironmentImpl) getActivityInfo(activityID, activityType
 
 func (env *testWorkflowEnvironmentImpl) cancelWorkflow() {
 	env.postCallback(func() {
-		// RequestCancelWorkflow needs to be run in main thread
-		env.RequestCancelWorkflow(env.workflowInfo.Domain,
-			env.workflowInfo.WorkflowExecution.ID,
-			env.workflowInfo.WorkflowExecution.RunID)
+		env.workflowCancelHandler()
```

Comment:
the old code would call onChildWorkflowCanceledListener() call

Reply:
let us wait on testsuite, i still have some issue dealing with it